landing-pages/site/data/commiters.json

```diff
@@ -0,0 +1,194 @@
+[
+  {
+    "name": "Maxime “Max” Beauchemin",
+    "nick": "mistercrunch",
+    "image": "/stock-guy.jpg",
+    "social_media": [
+      {
+        "icon": "icons/github.svg"
```

Comment:
I think it's better to separate the presentation layer from the content. We have a limited set of icons and we are able to predict what the socials will be. This will make writing documentation easier.

Reply:
done